devdocs/orientation-guide.md

```diff
@@ -0,0 +1,130 @@
+*   Committer: Significant contributions sustained over a long period of time.
+*   Core member: Significant and sustained contributions in recent times. Core members can progress through the levels: Snr Developer, Area Lead, Project Lead, and Project Manager.
+
+## Getting started
+
+Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resource when you cannot. To become a TEAMMATES contributor, you need to start honing those skills. To help you with that, we have created a sequence of tasks you can try to complete.  Try to complete as many of them as you can, in the order they are listed. Of course we are happy to guide you if you encounter any difficulties when doing these tasks; just post your question in [our issue tracker](https://www.google.com/url?q=https://github.com/TEAMMATES/teammates/issues&sa=D&ust=1458427634380000&usg=AFQjCNEfiA-pDqfeFk-NWy7uSU2vJdd4UA). If you are seeking help on project setup, remember to include the below in your post.
+
+1. Step in the setting up guide that you are in
+
+2. What measures you took to address the issue
+
+3. Operating system
+
+4. Screenshots (if any)
+
+
+Note: The task descriptions are brief by intention. We want you to try to figure out how to do those things by yourself.
+
+Orientation task list
+
+<mark style="background-color: #cfe2f3">**Phase A**: Know the project</mark>
+1.  Understand what TEAMMATES is about.
+    * [product intro page](https://www.google.com/url?q=https://teammatesv4.appspot.com&sa=D&ust=1458427634385000&usg=AFQjCNEDi3V_xRJj6Sv4x0rTuMreo4zIXA) shown to potential users
+
+    * [project vision, challenges, and principles](https://www.google.com/url?q=http://teammatesv4.appspot.com/dev/devman.html%23project&sa=D&ust=1458427634387000&usg=AFQjCNGm_T3iGMuSsidiGRVwLGI_Ybs4Gg)
+    * [feature overview](https://www.google.com/url?q=https://teammatesv4.appspot.com/features.html&sa=D&ust=1458427634388000&usg=AFQjCNFEaB-epSz6Mo5ICeLwqA-BNxCJMg) (users’ point of view)
+
+2.  Decide in which role(s) you want to contribute. i.e., tester, copy-editor, etc. as specified [earlier in this document](#id.49lgk6iu1w59).
+3.  If you are aiming for tech writer or tester roles, get an instructor account using the link in the TEAMMATES home page. Remember to mention the purpose of your request under any other comments.
+
+    If you are sure you want to become a contributor, you can skip this step. In Phase B, you get to set up TEAMMATES on your own machine and use that instance to try the functionality.
+
+    If you plan to contribute code, continue with the tasks below.
+
+
+
+<mark style="background-color: #cfe2f3">**Phase B**: Set up locally</mark>
+
+1.  Set up TEAMMATES development environment on your computer.  
+    Important:  
+    * Follow instructions to the letter. <mark>Install the specified versions of the tool stack, not the latest versions.</mark>  
```

Comment:
I dont think <mark> works in Github. Please check if there is another way to emphasize this line